### DISCUSSION TITLE
FR: Entry Edit - Entries Field Enhancement

### DISCUSSION THREAD

**Discussion by richhayler** [craftcms/cms#8661] [Ideas]
Selected entries for an **entries** field unecessarily take up too much vertical space.  

On the edit entry page within the control panel It would be great if selected entries (for an **entries** field type) were not listed _vertically_ but instead floated _horizontally_ (or there was a configurable option to specify this). Furthermore if the floated entries could then be sorted by dragging them horizontally to rearrange the order this would make for a better user experience.

While this issue is not too big of a problem for a single entry form, when an **entries** field is used within a matrix and there are multiple selected block entries and multiple matrix blocks it can make the entry edit page quite unwieldy.

While it is possible to override the control panel CSS to force selected entries (for an entries field) to be floated horizontally the ability to then rearrange them is lost.

I appreciate there are some upcoming changes to form layout options so it would be great if this one could be considered as part of that please.

**Comment by brandonkelly** [comment]
Actually planning on a version of this for 3.2 already, along the same lines as #2632. In the long run we are planning on merging the concepts of entries, categories, and tags (#1949), and to get there we first need to give Entries fields more UI options so they can behave like Categories and Tags fields.
